.github/workflows/coverage.yml

```diff
@@ -0,0 +1,116 @@
+name: Code coverage
+
+on:
+  push:
+    branches:
+      - main
+  workflow_dispatch:
+    inputs:
+      mode:
+        description: "Mode filter for era-compiler-tester"
+        required: true
+        default: "0.8.26"
```

Comment:
This will probably stop working when we update to 0.8.27 ><

Reply:
Oh, actually `0.8` works nice. I can use that.